source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -29,13 +29,16 @@ Use the :mc:`mc support perf` command to review the performance of the S3 API (r
 
 The resulting tests can provide general guidance of deployment performance under S3 ``GET`` and ``PUT`` requests and identify any potential bottlenecks.
 For more complete performance testing, consider using a combination of load-testing using your staging application environments and the MinIO `WARP <https://github.com/minio/warp>`_ S3 benchmarking tool.
-   
+
 :mc:`mc support perf` has the following subcommands
 
 #. :mc-cmd:`~mc support perf drive`
 
    Measure the speed of drives in a MinIO deployment.
 
+   :mc-cmd:`mc support perf drive` temporarily suspends S3 API calls during the test.
+   Incoming requests are queued until the test is complete or the command is cancelled.
+
```

Comment:
🤔  I debated making these admonitions, but they sure take up a lot of space in the page.

Reply:
I think that's the right call. Admonitions can be obnoxious.